spec/index.html

```diff
@@ -360,12 +359,23 @@ <h3>Triple Terms and Reification</h3>
           aria-label="An RDF graph containing a reifying triple where the triple term corresponds to an asserted triple"></object>
       </a>
       <figcaption id="fig-asserted-triple-term-alt">
-        An <a>RDF graph</a> containing a <a>reifying triple</a> where the <a>triple term</a> corresponds to an <a>asserted triple</a>.
+        An <a>RDF graph</a> containing a <a>triple annotation</a>, which is a <a>reifying triple</a> whose <a>triple term</a> corresponds to an <a>asserted triple</a>.
         The diagram represents the proposition as a fact using the asserted triple, meaning that the relationship holds.
       </figcaption>
     </figure>
 
-    <p>Note that a <a>triple term</a> may also have another <a>triple term</a> as an <a>object</a>.</p>
+    <p class="note">
+      A <a>proposition</a> denoted by a <a>triple term</a> is an abstract,
+      logical relationship; assertions are always made about some
+      <a>reifier</a> thereof. There can be multiple, distinct reifiers
+      related to the same abstract proposition, such as statements with
+      different sources, or circumstances with different characteristics.
```

Comment:
Attempting to take recent suggestions and comments into account:
```suggestion
      Since <a>triple terms</a> denote abstract, logical <a>propositions</a>,
      assertions involving them require the use of <a>reifying triples</a>. The
      <a>reifier</a> is then used for further description.
      There can be multiple, distinct reifiers related to the same abstract
      proposition, such as statements with different sources, or circumstances
      with different characteristics.
```

Reply:
Are we now saying, not only that reifiers are defined only by their presence as a subject in a triple with `rdf:reifies` as predicate and an explicit triple term as object, but also that an explicit triple term can only be found as the object of an `rdf:reifies` relationship? 

In other words, that non-`rdf:reifies` relationships that might appropriately be expressed in pseudo-turtle as — 
```
:b ex:rship << ex:sub ex:pred ex:obj >> .
``` 
— must instead be expressed as —
```
ex:rei rdf:reifies << ex:sub ex:pred ex:obj >> . 
:b     ex:rship    ex:rei .
```
?

Reply:
Good point. While:
```turtle
:b ex:rship << ex:sub ex:pred ex:obj >> .
```
does expand to:
```turtle
:b     ex:rship    _:rei .
_:rei rdf:reifies <<( ex:sub ex:pred ex:obj )>> . 
```
using a "bare" triple term like:
```turtle
:b ex:rship <<( ex:sub ex:pred ex:obj )>> .
```
is not forbidden (and there may be advanced uses for that, just as for nested triple terms).

So I propose this revision of the above reformulation:
```suggestion
      Since <a>triple terms</a> denote abstract, logical <a>propositions</a>,
      they are only allowed as objects of other triples.
      Assertions involving them should commonly use <a>reifying triples</a>;
      the <a>reifier</a> is then used for further description.
      There can be multiple, distinct reifiers related to the same abstract
      proposition, such as statements with different sources, or circumstances
      with different characteristics.
```
(Still solely focused on reification use, which is the topic of this section, but other than being explicit about the use as objects, avoiding prohibitive language. Cf. #218.)